coordinator/src/charm.py

```diff
@@ -217,7 +217,7 @@ def hostname(self) -> str:
         return socket.getfqdn()
 
     @property
-    def service_hostname(self) -> str:
+    def app_hostname(self) -> str:
```

Comment:
[the PR you reference in the issue](https://github.com/canonical/cos-coordinated-workers/pull/8/files#diff-f4d164a9eb0247693004815da595cd4f1b64fe88b2ac56378f2006082600184a), which adds `Coordinator.app_hostname`, was merged long ago. Are we not using it because it requires the Coordinator to be instantiated, and we need it before that?

Reply:
tried it out and yes, that's why.
Not seeing an elegant way to work around this for now.